pynitrokey/cli/nk3/__init__.py

```diff
@@ -467,6 +467,8 @@ def status(ctx: Context) -> None:
             local_print(f"Free blocks (int):  {status.ifs_blocks}")
         if status.efs_blocks is not None:
             local_print(f"Free blocks (ext):  {status.efs_blocks}")
+        if status.variant is not None:
+            local_print(f"Variant:            {status.variant.name}")
```

Comment:
Eh, too late for that now, but this kind of formatting should be regulated through field width, instead of spaces hardcoding.